app/api/helpers/scheduled_jobs.py

```diff
@@ -29,13 +29,6 @@ def send_after_event_mail():
     from app import current_app as app
     with app.app_context():
         events = Event.query.filter_by(state='published', deleted_at=None).all()
-        upcoming_events = get_upcoming_events()
```

Comment:
If `get_upcoming_events` is not used anywhere else, comment on it saying `TODO: Unused function. Remove`

Reply:
Done